pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, trainer):
 
         # used to validate checkpointing logic
         self.has_trained = False
```

Comment:
```suggestion
        self._has_trained = False
```
unrelated to this pr, but I would consider also this as protected...

Reply:
Done